CONTRIBUTING.md

```diff
@@ -1,18 +1,102 @@
+
+[![POEditor](https://poeditor.com/public/images/logo_small.png)](https://poeditor.com/join/project/IE4GC127Ki)
 
 Before each big release, translations are pulled in from POEditor.
 
-## Map style
+## Helping mappers by solving their notes
+
+- [x] Requires no programming knowledge!
+- [ ] Requires no OpenStreetMap knowledge!
+
+As you probably noticed, you can choose "Cannot answer" in StreetComplete and thus leave a note by
+
+You can help with [processing OSM notes opened by StreetComplete users](https://ent8r.github.io/NotesReview/?query=StreetComplete&limit=100&start=true). (You can also do it without that special tool, but it helps.) When doing so, you can also find systematic problems or misunderstandings of user's when solving StreetComplete quests.
+
+If you find such user experience problems, please report them back in the [issue tracker of StreetComplete](https://github.com/westnordost/StreetComplete/issues). Do not forget to add links to examples, where you found them.
+
+## Joining discussions
+
+- [x] Requires no programming knowledge!
+- [x] Requires no OpenStreetMap knowledge!
+
+You can always join in discussions in issues for new quests or similar stuff. Often local knowledge or so is required, so we can estimate whether a quest can be asked in different regions.
+OpenStreetMap (and StreetComplete) works, because of a diverse community all over the world.
+
+[Especially look at the issues tagged with "help wanted"](https://github.com/westnordost/StreetComplete/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
+
+## Testing and reporting issues
+
+- [x] Requires no programming knowledge!
+- [x] Requires no OpenStreetMap knowledge!
+
+If you experience problems, crashes or a quest is not clear to you, feel free to open an issue for that. Remember to open one issue _for one matter_, so do not open one issue "I have found several problems", but one for each problem.
+
+1. If you have questions, remember to [read the FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ) first.
+2. Look [whether your issue has already been reported](https://github.com/westnordost/StreetComplete/issues).
+3. Now you can open a new issue. Give it a meaningful title and describe the problem. Do mention the StreetComplete and Android version, if it is a technical bug.
+
+### Map style issues
+
+Note that the map style used by StreetComplete is developed as a [separate project](https://github.com/ENT8R/streetcomplete-mapstyle), so please report issues related to it there.
+
+### Suggesting new quests
+
+We appreciate all quest ideas, you and others have, but some ideas cannot be added into the app, due to various design decicions and requirements. This is required in order to make a great app for all users of StreetComplete, whose contributions actually help the OpenStreetMap project. A quest, which nobody can solve and where users potentially choose wrong options and add wrong data to OpenStreetMap hurts not only OSM, but also this app.
```

Comment:
Tx for the typo. Fixed.
As for the commas: I removed them as they are indeed not correct, AFAIK. Only the last one is required grammatically AFAIK. It's a simple "but" subclause.

Reply:
But the English language uses way less commas than e.g. German at all, so this may be really just removed...

Reply:
Well… maybe, but rather than a general statement a reference to some grammar rule may help more here. I can hardly remove all commas, because it may use less. :smile: 

While we are at it: @goldfndr are you a native speaker of English maybe? Such a review would help a bit more, and mention that really helps in this case. :smiley: